src/editor/SegmentationSidebar.js

```diff
@@ -42,7 +42,7 @@ const SegmentationSidebar = ( { onMetaFieldChange, selected_segment_id } ) => {
 				href="/wp-admin/admin.php?page=newspack-popups-wizard#/segmentation"
```

Comment:
Need to update this link to `#/segments` now that we've changed the route in https://github.com/Automattic/newspack-plugin/pull/833.

Reply:
Nice spot! Fixed in https://github.com/Automattic/newspack-popups/pull/431/commits/a48c8d91379e39f05719be3f2ae4f115b6aebe62